lib/routes/one/news.ts

```diff
@@ -0,0 +1,87 @@
+import { Route, Data, DataItem } from '@/types';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+
+const apiUrl = 'https://wufazhuce.com/';
+const NAME = '「ONE · 一个」';
+
+export const route: Route = {
+    path: `/news`,
+    categories: ['reading'],
+    example: '/one/news',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['wufazhuce.com'],
+            target: '/news',
+        },
+    ],
+    name: NAME,
+    maintainers: ['sicheng1806'],
+    handler,
+};
+
+async function handler(): Promise<Data> {
+    const resp = await got(apiUrl);
+    const $ = load(resp.body);
+    let items: DataItem[] = [
+        ...$('#carousel-one div.item')
+            .toArray()
+            .map((item) => {
+                const a = $(item).find('.fp-one-cita a').first();
+                return {
+                    title: `[摄影]${a.text()}`,
+                    link: String(a.attr('href')),
+                    description: '',
+                };
+            }),
+        ...$('.fp-one-articulo a')
+            .toArray()
+            .map((item) => {
+                const a = $(item);
+                return {
+                    title: `[ONE 文章]${a.text()}`,
```

Comment:
Category should be placed inside [`category`](https://github.com/DIYgod/RSSHub/blob/a41cfc79ec93186793bf5b98a1c46cbd9a03edd3/lib/types.ts#L37) instead of `title`.